src/atomate2/vasp/sets/BaseMPGGASet.yaml

```diff
@@ -111,13 +111,15 @@ INCAR:
 KPOINTS:
   reciprocal_density: 64
   reciprocal_density_metal: 200
-POTCAR_FUNCTIONAL: PBE
+POTCAR_FUNCTIONAL: PBE_54
```

Comment:
I don't think this should be changed to keep the input sets consistent with pymatgen?

Reply:
We'll bring this up with @kristinpersson when we discuss potential input set changes but I think the idea was to break backwards compat in favor of more future-proof settings if we decide to do a complete rerun of all non-deprecated MP calcs.

Reply:
It seems like a large omission if atomate2 is unable to generate calculations compatible with current MP data. Even with refreshing the input set, presumably this should be a new, additional configuration? Especially if MP has not yet released a new database version with the new settings.

Reply:
I agree with @mkhorton here.

Reply:
Thanks all for taking a look! Reverted in 36c25294.